airflow/providers/microsoft/azure/operators/container_instances.py

```diff
@@ -319,6 +319,9 @@ def _monitor_logging(self, resource_group: str, name: str) -> int:
                 if state in ["Running", "Terminated", "Succeeded"]:
                     try:
                         logs = self._ci_hook.get_logs(resource_group, name)
+                        if len(logs) > 0 and logs[0] is None:
```

Comment:
```suggestion
                        if logs > 0 and logs[0] is None:
```

`len` might not be necessary

Reply:
Unfortunately that change causes a type error:

```
Traceback (most recent call last):
  File "/home/azureuser/pyenv-airflow/lib/python3.8/site-packages/airflow/providers/microsoft/azure/operators/container_instances.py", line 321, in _monitor_logging
    if logs > 0 and logs[0] is None:
TypeError: '>' not supported between instances of 'list' and 'int'
```

Hence len seems to be needed.

Reply:
```suggestion
                        if logs and logs[0] is None:
```

sorry, should remove the `> 0` as well

Reply:
Yeah that works and looks cleaner, thanks!